Adding documentation for HTTPS

nikhiljindal opened the pull request:
Adding documentation for HTTPS ingresses

cc @csbell @G-Harmon

nikhiljindal commented:
Had pushed the commit with wrong author :)
Fixed it now

G-Harmon commented on examples/zone-printer/https.md:
```diff
@@ -0,0 +1,54 @@
+While a single cluster ingress also supports blocking HTTP traffic all together,
+kubemci does not support it yet.
+
+https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/issues/142 has
```
"#142 is the feature request for this" ?

nikhiljindal replied:
updated

G-Harmon commented on examples/zone-printer/https.md:
```diff
@@ -0,0 +1,54 @@
+
+In addition, updating the secret is not yet supported.
+
+https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/issues/141 has
```
"#141 has more details and includes a workaround" ?

nikhiljindal replied:
Updated

nikhiljindal commented:
Thanks for the review @G-Harmon 
Updated code as per comments.

Also updated ingress-gce documentation to ensure that it has instructions for both pre-shared cert annotation and secret: https://github.com/kubernetes/ingress-gce/pull/144

nikhiljindal commented:
cc @prameshj  as FYI
